### PR TITLE
117 open data export

### PR DESCRIPTION
**Pull request by caseysmithpgh**
Here you'll find I updated the syntax to fit SELECT INTO approach. All row counts seem to align on my local run.

### CODE REVIEW

File: sql/opendata.sql [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+c.nccother,
+p.noncitycost AS totalnoncityplannedcommit,
+p.totalcost AS totalplannedcommit,
+p.magencyacro, 
+p.magencyname, 
+c.ccpversion
+FROM cpdb_commitments as c,
+    cpdb_projects as p
+WHERE c.maprojid = p.maprojid)
+SELECT
```
Comment (td928) [inline sql/opendata.sql:29]: Hey @caseysmithpgh do want to point out that the final opendata commitment table does not have a same order of columns as the data dictionary. Don't know if this needs to be addressed before final exporting.

Comment (caseysmithpgh) [reply sql/opendata.sql:29]: @td928 ah, yes good catch. I'm now noticing this is the case for the commitments table, Would this need to be corrected under the second select statement, scrapping c.*, and instead calling them in the correct order? Or is there a less lengthy approach you can think of?

Comment (mbh329) [reply sql/opendata.sql:29]: can you call the `alter_table` function and set the column order? https://stackoverflow.com/questions/285733/how-do-i-alter-the-position-of-a-column-in-a-postgresql-database-table#:~:text=PostgreSQL%20currently%20defines%20column%20order,you%20reach%20the%20desired%20layout.

Comment (mbh329) [reply sql/opendata.sql:29]: There might be a more slick way to do this though @AmandaDoyle @td928 might have some better ideas

Comment (AmandaDoyle) [reply sql/opendata.sql:29]: Order of the fields does matter.  You can call all the fields in one shot and do the join this way:
```
FROM cpdb.cpdb_commitments as c
LEFT JOIN cpdb.cpdb_projects as p ON c.maprojid = p.maprojid
LEFT JOIN cpdb.cpdb_budgets as b ON c.budgetline = b.budgetline AND c.maprojid = b.maprojid;
```

**Review by AmandaDoyle — CHANGES_REQUESTED** [review]
See my one comment on how to recast the join for creating the commitments table.  Everything else looks good.

**Comment by mbh329** [comment]
Check order of columns in export:

- [ ] cpdb_opendata_commitments 
- [ ] cpdb_opendata_projects
- [ ] cpdb_opendata_projects_pts
- [ ] cpdb_opendata_projects_poly

**Comment by mbh329** [comment]
In regards to column order, all the files look good to me. I also took a look at the % of records that were mapped (projects_poly + projects_pts/total projects) and the numbers are consistent (~40% of projects successfully mapped) with what we have seen in the past. will do a couple more spot checks but think this is good to go

**Comment by mbh329** [comment]
This looks great! I went a little crazy with the checks and tested the github action on the feature branch and the outputs look good and are succesfully uploaded to DO! Great job @caseysmithpgh, this is ready to be merged in. You do have a merge conflict atm because there were some recent merges into this file on our end so you will have to deal with those before merging

**Review by mbh329 — APPROVED** [review]

**Review by AmandaDoyle — APPROVED** [review]

**Pull request merged**